src/frontend/efiling-frontend/src/domain/package/package-review/DocumentList.js

```diff
@@ -1,12 +1,15 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { MdHelp } from "react-icons/md";
 
 import { errorRedirect } from "../../../modules/helpers/errorRedirect";
+import { withdrawTooltipData } from "../../../modules/test-data/withdrawTooltipData";
```

Comment:
Should this static content live in test-data? It looks to be used on the actual screen, not just in the tests.